/pgv100_node:
  ros__parameters:
    pgv:
      port_name: "/dev/ttyACM0" # Port name of left motor controller serial port.
      baudrate: 115200 # Baudrate of left motor controller serial port. 
      serial_port_group: "/dev" # Serial port group to detect tty
      id_vendor: "Roboteq" # ID Vendor to detect tty (Front Motor Controller udevadm info)
      id_vendor_id: "20d2" # ID Vendor ID to detect tty (Front Motor Controller udevadm info)
      id_model_id: "5740" # ID Model ID to detect tty (Front Motor Controller udevadm info)
      id_path: "pci-0000:00:14.0-usb-0:4:1.0" # ID Path to detect tty (Front Motor Controller udevadm info)
      timeout: 1000 # Timeout to open serial port (ms)
